### ISSUE TITLE
Tester doesnt errors out

### ISSUE THREAD

**Issue by idkjs** [kgoggin/reason-react-apollo#4]
Hi @kgoggin. I am again the guinea pig!

Just trying running this and getting this error in your [packages/tester](https://github.com/kgoggin/reason-react-apollo/tree/master/packages/tester)

```sh
➜  tester yarn build
yarn run v1.17.3
$ yarn run generate && bsb -clean-world -make-world
$ graphql-codegen
  ✔ Parse configuration
  ❯ Generate outputs
    ❯ Generate ./generated/GraphQLTypes.re
      ✔ Load GraphQL schemas
      ✔ Load GraphQL documents
      ✖ Generate
        → Unable to load template plugin matching graphql-codegen-reason-c
…
    ❯ Generate ./generated/Graphql.re
      ✔ Load GraphQL schemas
      ✔ Load GraphQL documents
      ✖ Generate
        → Unable to load template plugin matching graphql-codegen-reason-r
…


 Found 2 errors

  ✖ ./generated/GraphQLTypes.re
    Unable to load template plugin matching 'graphql-codegen-reason-client
'.
    Reason: 
      Cannot find module '/Users/prisc_000/Downloads/reason-react-apollo-m
aster/node_modules/graphql-codegen-reason-client/dist/index.js'. Please ve
rify that the package.json has a valid "main" entry

  ✖ ./generated/Graphql.re
    Unable to load template plugin matching 'graphql-codegen-reason-react-
apollo'.
    Reason: 
      Cannot find module '/Users/prisc_000/Downloads/reason-react-apollo-m
aster/node_modules/graphql-codegen-reason-react-apollo/dist/index.js'. Ple
ase verify that the package.json has a valid "main" entry


Something went wrong
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
➜  tester 
```

I got the same error on another set up and figured i was doing something wrong. There is, in fact, no "main" entry, as far as I can tell. Ideas?

Thank you.

**Comment by kgoggin** [comment]
Yeah, this is a problem with the package that got published, I'm ashamed to admit. Still figuring out the best/right way to use lerna. I'll get a fix published ASAP - sorry about that!

**Comment by kgoggin** [comment]
Should be fixed in v1.1.0!
